Bootstrap app after forking

CloudNiner opened the pull request:
## Overview

Updates:
- Dependencies in package.json
- Dependencies in bower.json
- Deployments to both a staging and production environment
- Instructions for how to create an environment in AWS along with `./scripts/infra` for regular deployment

## Testing

Follow setup instructions in "Development" section of README. Ensure you're able to run the dev site and it looks like MuseumStat. 

Proof of deployment is that the following sites exist, use HTTPS and work:
- http://staging.impactview.org
- http://impactview.org

Commit:
Update top level project refs to ImpactView

Commit:
Update npm package dependencies

Replaced connect.serve with servestatic based on:

https://stackoverflow.com/a/33113395

due to updates in the `grunt-contrib-connect` package

Commit:
Update bower packages

Commit:
Add package-lock.json

Commit:
Update travis.yml for this project

Commit:
Create staging and production deployments

Using CloudFront with SSL. Instructions in README
provide instructions for how they were set up.

Can access site now at both:
- staging.impactview.org
- impactview.org

Commit:
Revert "Update bower packages"

This reverts commit 6288f4687291c3e32ba6fa8181d0429b1298aaf6.

Trouble in paradise

Commit:
Update easy-to-update bower dependencies

Skipped updates to:
- leaflet-draw: Introduces change in dep tree that modifies the
  build such that it throws an error because bower now tries
  to include leaflet-src.js and we don't want that since its
  provided by cartodb.js
- lodash (v4+): pluck replaced with map, and the straightforward
  substitution in their CHANGELOG didn't work, throwing
  additional errors.

Commit:
Satisfy linter

Commit:
Copy example config.js in travis

Commit:
QOL tweaks to scripts/infra

KlaasH commented on app/index.html:
```diff
@@ -55,8 +55,8 @@
     <!-- bower:js -->
-    <script src="bower_components/jquery/dist/jquery.js"></script>
     <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/jquery/dist/jquery.js"></script>
```
Was having jQuery first causing something to fail?  This way means AngularJS will be using its own jqLite for `$element`, right?  Which is fine, except in those occasional instances where one wants to do something unusual with `$element` and having the full jQuery comes in handy.

CloudNiner replied:
Hmm. Good eye. I didn't manually change this, the order was modified by the package updates I did. I'll see about reverting it.

KlaasH commented on app/scripts/acs/acs-graphs.js:
```diff
@@ -61,27 +61,27 @@
+                        var percent = numberFilter(d.data.value / total * 100, 1);
+                        return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
+                                ' (', percent, '%)'].join('');
+                    });
```
These are identical, right?  Moving them right below the conditional would work the same and only require one copy.

KlaasH commented on app/scripts/acs/acs-graphs.js:
```diff
@@ -106,14 +106,14 @@
+                    var percent = $filter('number')(d.data.value / total * 100, 1);
+                    return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
+                            ' (', percent, '%)'].join('');
+                });
```
Here it is again.  Now it seems like a helper function is in order (moving the one above outside the conditional still makes sense, even though it'll be shorter).

KlaasH commented on .travis.yml:
```diff
@@ -6,9 +6,7 @@ cache:
-  - '0.12'
-  - '4.2'
-  - '5.1'
+  - '6'
```
The README still says "nodejs 0.12+".  Should that be changed?

KlaasH reviewed (approved):
A few notes and questions below, but it provisioned (after I installed ruby-dev and ruby 2.0--I was on 1.9.1, which `sass` didn't like) and came up fine.

:+1:

Commit:
DRY tooltip HTML generation in acs graphs

Commit:
Update required node version in README

Commit:
Load jQuery first

So Angular takes advantage of it everywhere.

Easiest way to do this was to specify jquery
as an explicit project dependency, and let
wiredep handle modification of the load order.

I don't see a quick, clean way digging into
wiredep settings to manually swap the order
without including jquery as a dep.
